benefits/static/css/styles.css

```diff
@@ -312,6 +315,11 @@ footer .footer-links li a:visited {
   border-width: 2px;
 }
 
+/* Make `button` elements look similar to button-links. */
+button:focus {
+  outline: none !important;
+}
```

Comment:
Remove this

Reply:
For more context, @machikoyasuda was showing me that we already have `<button>`s on the site that have the light-blue outline, which is different from `<a>` elements styled to look like buttons, which don't have the light-blue outline.

The "Espanol" button and form submission buttons are good examples.

I added this rule just because it was really obvious between the index page (which has a `<button>` CTA) vs. the agency_index page (`<a>` CTA) that the "button" outlines were different, but I'm fine with removing this and not worrying about that difference.